appengine-java8/spanner/pom.xml

```diff
@@ -22,16 +22,23 @@
   <artifactId>appengine-spanner-j8</artifactId>
   <version>1.0-SNAPSHOT</version>
   <packaging>war</packaging>
-  <properties>
-    <failOnMissingWebXml>false</failOnMissingWebXml>
-  </properties>
 
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
   <parent>
-    <artifactId>appengine-java8-samples</artifactId>
-    <groupId>com.google.cloud</groupId>
-    <version>1.0.0</version>
-    <relativePath>..</relativePath>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.8</version>
   </parent>
+
+  <properties>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+
+    <failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
Shouldn't be required if we are using the correct version of the maven war plugin.